src/lib/progress-bar/progress-bar.ts

```diff
@@ -113,6 +131,21 @@ export class MatProgressBar extends _MatProgressBarMixinBase implements CanColor
       return {transform: `scaleX(${scale})`};
     }
   }
+
+  /** Attaches value bar element animation end listener. */
```

Comment:
I would remove this comment since it doesn't always describe what `ngAfterViewInit` does. Really the best thing would be to add another method like `_addAnimationEndListener` and call it here, but I don't feel that strongly